src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -35,55 +35,81 @@ export const writePackageTransform = (options: NgPackagrOptions) =>
     ];
 
     if (!ngEntryPoint.isSecondaryEntryPoint) {
-      const assetFiles: string[] = [];
-
       // COPY ASSET FILES TO DESTINATION
       spinner.start('Copying assets');
 
       try {
-        for (const asset of [...ngPackage.assets, 'LICENSE', '**/README.md']) {
-          let assetFullPath = path.join(ngPackage.src, asset);
-
+        type AssetCopyTask = [srcPath: string, destPath: string];
+        const assetCopyTasks: AssetCopyTask[] = [];
+
+        // Parse string assets and add to tasks
+        const assetGlobs = ngPackage.assets
```

Comment:
The variable name here is misleading, since these can be globs and non globs.

Reply:
Hi. I'm not really sure about "non globs". If that means filenames, I guess filenames are also valid globs?

Reply:
Yes that is correct